src/__test__/pages/experiments/[experimentId]/data-exploration/components/differential-expression-tool/__snapshots__/DiffExprManager-regression-load-on-second-render.test.jsx.snap

```diff
@@ -13,9 +13,9 @@ exports[`DiffExprManager regression test -- diff exp would not reload after \`go
 Object {
   "payload": Object {
     "cellSets": Object {
-      "basis": "all",
-      "cellSet": "cluster-2",
-      "compareWith": "rest",
+      "basis": "cluster-a;",
+      "cellSet": "condition-treated",
+      "compareWith": "condition-control",
```

Comment:
shouldn't this actually be formatted as `group/cluster-key` as we spoke about? Or is this the snapshot for the DifferentialExpression task that is being sent to the worker?

Reply:
updated in commit 5bbaf2a